ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Initiator.java

```diff
@@ -451,12 +451,13 @@ private boolean isEligibleForCompaction(CompactionInfo ci,
       ShowCompactResponse currentCompactions, Set<String> skipDBs, Set<String> skipTables) {
     try {
       if (skipDBs.contains(ci.dbname)) {
-        LOG.debug("Skipping {}::{}, skipDBs:{}", ci.dbname, ci.tableName, skipDBs);
+        LOG.info("Skipping {}::{}, skipDBs::size:{}", ci.dbname, ci.tableName, skipDBs.size());
         return false;
       } else {
         if (replIsCompactionDisabledForDatabase(ci.dbname)) {
           skipDBs.add(ci.dbname);
-          LOG.debug("Skipping {}::{}, skipDBs:{}", ci.dbname, ci.tableName, skipDBs);
+          LOG.debug("Skipping {} as compaction is disabled due to repl; skipDBs::size:{}",
```

Comment:
I think this log should definitely be info-level

Reply:
Sure. Fixed in latest commit.